features/support/esi_fund_helpers.rb

```diff
@@ -32,15 +32,7 @@ def withdraw_esi_fund(*args)
     withdraw_document(:esi_fund, *args)
   end
 
-  def check_header_metadata_depth_is_limited(slug, depth: 1)
-    published_document = RenderedSpecialistDocument.find_by_slug(slug)
-
-    headers = published_document.details.fetch("headers")
-
-    expect(find_header_depths(headers).max).to eq(depth)
-  end
-
-  def document_body_with_deeply_nested_headers
+  def document_body
```

Comment:
Given these are no longer doing anything special with the headers at all, the body no longer need to be defined separately in a helper - they should be inline as with the other document types (aaib, raib etc).